info/guide/guide-chapter-3.md

```diff
@@ -932,3 +932,19 @@ if (cond1 && (cond2 || cond3)) {
     doSomething()
 }
 ```
+#### <a name="r3.16.2"></a> 3.16.2 Too complex conditions
+Too complex conditions should be replaced with simplified ones.
```

Comment:
Please specify more explicitly, that we are talking about boolean algebra here, not about extracting conditions into variables

Reply:
Fixed